### ISSUE TITLE
[New Grapher] Save as pickled figure for later editing

### ISSUE THREAD

**Issue by roboguy222** [McDermott-Group/servers#6]

**Comment by amopremcak** [comment]
Would you like this in the form of an additional button? Also, how would you display the figure object @roboguy222. Since this is not a pyplot object, you need to provide a canvas for this figure to embed itself on. Perhaps the pickle button should map the figure object onto a corresponding pyplot object and then pickle it. Thoughts?

**Comment by roboguy222** [comment]
Yeah, I guess an additional button that brings up a save dialog/directory window or something would be fine.  It is probably ok to just pickle the fig.  We can then use pyplot or whatever to show that figure later (test this).  As long as we can go back and edit things, it doesn't really matter.
